frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -95,99 +309,93 @@ export function CopyableCodeSnippet({ code, onCopy }: { code: string; onCopy?: (
-
-  const [activeTabKey, setActiveTabKey] = useState("download");
-  const [includeVolumeData, setIncludeVolumeData] = useState(true);
-  const [keepWindowOpen, setKeepWindowOpen] = useState(false);
-  const [startedExports, setStartedExports] = useState<string[]>([]);
-  const [selectedLayerName, setSelectedLayerName] = useState<string | null>(null);
-  const [selectedBoundingBoxID, setSelectedBoundingBoxId] = useState(-1);
-
+function _DownloadModalView({
+  isOpen,
+  onClose,
+  initialTab,
+  initialBoundingBoxId,
+}: Props): JSX.Element {
   const activeUser = useSelector((state: OxalisState) => state.activeUser);
   const tracing = useSelector((state: OxalisState) => state.tracing);
   const dataset = useSelector((state: OxalisState) => state.dataset);
-  const userBoundingBoxes = useSelector((state: OxalisState) =>
+  const rawUserBoundingBoxes = useSelector((state: OxalisState) =>
     getUserBoundingBoxesFromState(state),
   );
   const isMergerModeEnabled = useSelector(
     (state: OxalisState) => state.temporaryConfiguration.isMergerModeEnabled,
   );
-  const activeMappingInfos = useSelector(
-    (state: OxalisState) => state.temporaryConfiguration.activeMappingByLayer,
+  const hasVolumeFallback = tracing.volumes.some((volume) => volume.fallbackLayer != null);
+
+  const [activeTabKey, setActiveTabKey] = useState<TabKeys>(initialTab ?? "download");
+  const [includeVolumeData, setIncludeVolumeData] = useState(true);
+  const [keepWindowOpen, setKeepWindowOpen] = useState(true);
+  const [selectedLayerName, setSelectedLayerName] = useState<string>(
+    dataset.dataSource.dataLayers[0].name,
   );
 
   const layers = getDataLayers(dataset);
 
-  const selectedBoundingBox = userBoundingBoxes.find((bbox) => bbox.id === selectedBoundingBoxID);
-  let boundingBoxCompatibleInfo = null;
-  if (selectedBoundingBox != null) {
-    boundingBoxCompatibleInfo = isBoundingBoxExportable(selectedBoundingBox.boundingBox);
-  }
+  const selectedLayer = getLayerByName(dataset, selectedLayerName);
+  const selectedLayerInfos = getExportLayerInfos(selectedLayer, tracing);
+
+  const userBoundingBoxes = [
+    ...rawUserBoundingBoxes,
+    {
+      id: -1,
+      name: "Full dataset",
+      boundingBox: computeBoundingBoxFromBoundingBoxObject(selectedLayer.boundingBox),
+      color: [255, 255, 255],
+      isVisible: true,
+    } as UserBoundingBox,
+  ];
+  const [selectedBoundingBoxId, setSelectedBoundingBoxId] = useState(
+    initialBoundingBoxId ?? userBoundingBoxes[0].id,
+  );
+
+  const datasetResolutionInfo = getDatasetResolutionInfo(dataset);
+  const { lowestResolutionIndex, highestResolutionIndex } = selectedLayerInfos;
+  const [rawResolutionIndex, setResolutionIndex] = useState<number>(lowestResolutionIndex);
+  const resolutionIndex = clamp(lowestResolutionIndex, rawResolutionIndex, highestResolutionIndex);
+  const [exportFormat, setExportFormat] = useState<ExportFormat>(ExportFormat.OME_TIFF);
```

Comment:
is ome tiff widely supported and thus a good default?

Reply:
Yes, I think so. We should probably track usage over time.